### DISCUSSION TITLE
Linearmouse trackpad speed slower on macOS 15 beta 2

### DISCUSSION THREAD

**Discussion by MoltenMonster** [linearmouse/linearmouse#755] [Q&A]
I forget what numbers I had set in Linearmouse before installing the beta but after installing it, the trackpad cursor speed slowed down. I spent some time trying to troubleshoot it and couldn't get the mouse speed I had before, even with max settings, and I'm wondering if the scales have changed, are these the max numbers for anyone else? Has anyone else had issues on this or any ideas on make my trackpad go even faster?
<img width="640" alt="Screenshot 2024-06-25 at 7 09 17 AM" src="https://github.com/linearmouse/linearmouse/assets/42647030/49c2ac58-3465-4a96-83b4-68ce0e9d7e91">

**Comment by pawelkontek** [comment]
Having the same issue.
